container/Containerfile

```diff
@@ -13,8 +13,11 @@ FROM alpine
 RUN apk add python3
 RUN addgroup --system ovirtimg
 RUN adduser --system --ingroup ovirtimg ovirtimg
-RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio
+RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio /images
 COPY --from=build /venv /venv
 COPY conf.d /etc/ovirt-imageio/conf.d
+COPY file.json file.json
+COPY disk.img /images
```

Comment:
I will change it to `VOLUME` or `CMD --mount` later. I thought it would be nice to copy the image to have a ticket with well-known size and name. But the ticket size can be modified before invoking `make container`, and mounting the image with `VOLUME` allows for a more portable (smaller) container image.

Reply:
The ticket and the image should not be part of the container.

In k8s the ticket will be provided by configmap, same way engine details will be provided. The disk will be provided via a pvc.